spec/identity_doc_auth/acuant/request_spec.rb

```diff
@@ -152,5 +152,58 @@
         expect(response.exception).to be_a(Faraday::ConnectionFailed)
       end
     end
+
+    context 'when the request resolves with a handled http error status' do
+      before do
+        allow(subject).to receive(:handle_http_response) do |http_response|
+          http_response
+        end
```

Comment:
This is super weird to me... since `Acuant::Request` is an abstract class

Maybe we just make a basic subclass for this test and use that instead?

```ruby
class SimpleAcuantRequest < IdentityDocAuth::Acuant::Request
  def handle_http_response(http_response)
    http_response
  end
end
```

Reply:
With a quick check, it's actually not needed, it seems, other than a few tests. I have removed it most of the places locally. I'll see about a different implementation.

Reply:
Yeah, I didn't have the brain to  dig into it yesterday...that's definitely a strange pattern. I cleaned it up and removed all those calls. The simple test implementation of the class was a good idea. Thanks!